### DISCUSSION TITLE
I have a failure that don't understand the reason

### DISCUSSION THREAD

**Discussion by yasinkarax** [community/community#145660] [Programming Help] [answered]
Can you explain what is the issue here?
![Screenshot_26-11-2024_184910_github com](https://github.com/user-attachments/assets/f4f8777d-df2c-46c7-9538-568f4cd61a05)
![Screenshot_26-11-2024_184847_github com](https://github.com/user-attachments/assets/89b66cc0-33d1-4d3b-bcc4-d35d67d33093)

**Comment by samus-aran** [comment, marked as answer]
<img width="649" alt="Screenshot 2024-11-29 at 11 08 58" src="https://github.com/user-attachments/assets/637a336d-668f-4177-8a93-968bc4e11a12">

Click on the 3/5 from your [commit history](https://github.com/yasinkarax/ToDo/commits/main/) and you should then further explore why 2 checks have failed (suspect there was a failure on the platform your deploying to).

**Reply by yasinkarax** [reply]
thanks
